CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.3.1
+
+### 🐞 Bug fixes
+* Increase strictness of the style API validation ([#10779](https://github.com/mapbox/mapbox-gl-js/pull/10779))
+* Remove strictly-increasing requirement for fog range validation ([#10772](https://github.com/mapbox/mapbox-gl-js/pull/10772))
```

Comment:
I think we can remove the spec-only changes as they are in the other changelog. The second one is more debatable as it touches a little bit of code in the fog culling, but seems mainly spec focused overall.
```suggestion
```

Reply:
👍 Yeah, I wondered about that. Sounds good.